apps/docs/content/guides/auth/auth-hooks.mdx

```diff
@@ -602,15 +602,15 @@ grant execute
 
 revoke execute
   on function public.custom_access_token_hook
-  from authenticated, anon;
+  from authenticated, anon, public;
 
 grant all
   on table public.profiles
   to supabase_auth_admin;
 
 revoke all
   on table public.profiles
-  from authenticated, anon;
+  from authenticated, anon, public;
```

Comment:
There is no need to revoke from the tables on `public`. However we do this for consistency with function revocation.